plugins/test/unit/plugins/test_registry.py

```diff
@@ -1,15 +1,21 @@
 from cStringIO import StringIO
 import json
 import os
+import shutil
 import tempfile
-import unittest
+
+try:
+    import unittest2 as unittest
```

Comment:
I addressed that in this PR: https://github.com/pulp/pulp/pull/1772

``unittest2`` will be required on EL6, but not for EL7 for F*. I added it to the spec file for pulp-devel on el6, and to the puppet class used by our test automation.